qiime2/sdk/proxy.py

```diff
@@ -95,6 +114,10 @@ def validate(self, level=NotImplemented):
 class ProxyArtifact(ProxyResult):
     """This represents a future Artifact that is being returned by a Parsl app
     """
+    def _view(self, view_type, recorder):
+        return self._get_element_(
```

Comment:
I think this and `view()` should be calling `self.result()` right? Then we alias. Whereas right now, it'll be a view of the non-aliased version.